packages/tests/esm-misc/src/entities/codegen/AuthorCodegen.ts

```diff
@@ -163,22 +163,66 @@ export abstract class AuthorCodegen extends BaseEntity<EntityManager, string> im
     return getField(this, "updatedAt");
   }
 
+  /**
+   * Partial update taking any subset of the entities fields.
+   * Unlike `set`, null is used as a marker to mean "unset this field", and undefined
+   * is left as untouched
```

Comment:
Period after "untouched". And if this is meant as the end of a paragraph, i.e. "Collections" is on a new line, maybe also do a new line between, like:

```
 * is left as untouched.
 *
 * Collections are exha...
```

Assuming the intent was for "Collections" to be a new paragraph (which is fine either way, just wasn't sure intent)